tests/test_banidb.py

```diff
@@ -1,12 +1,20 @@
 # -*- coding: utf-8 -*-
-import requests
 import json
-import banidb
+
+import pytest
+import requests
 from requests_mock.mocker import Mocker
 
+import banidb
+
 url = 'https://api.banidb.com/v2'
 
 
+@pytest.fixture
+def clear_cache():

```

Comment:
For deterministic output, cache should be cleared before running the test. There should be additional tests for testing the caching.